tests/integration/cocoon/test_seed_sample_data.py

```diff
@@ -0,0 +1,276 @@
+import unittest
+from pathlib import Path
+import pandas as pd
+import lusid
+from lusidtools.cocoon.seed_sample_data import seed_data
+from lusidtools.cocoon.utilities import load_json_file
+from lusidtools.cocoon.utilities import create_scope_id
+import logging
+import os
+import datetime
+import pytz
+import random
+
+logger = logging.getLogger()
+
+
+class CocoonSeedDataTestsBase(object):
+
+    """
+    Class description:
+    ------------------
+    This class creates the tests we use in other classes below.
+    We split the the file loading and tests into seperate classes for performance reasons.
```

Comment:
typo "separate"

Reply:
I don't think this is so much about performance as is readability and reuse. Its probably more useful to a user to tell them how to use it: subclass this base class and implement the Setup to populate LUSID, the tests then run against the data created

Reply:
Thanks, done.